crates/stackable-operator/src/commons/affinity.rs

```diff
@@ -85,6 +85,8 @@ pub fn affinity_between_role_pods(
             namespace_selector: None,
             namespaces: None,
             topology_key: TOPOLOGY_KEY_HOSTNAME.to_string(),
+            match_label_keys: None,
```

Comment:
question: Were these fields added in K8s `1.29`? Setting them to `None` should be fine then.

Reply:
Yep, https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#matchlabelkeys